### DISCUSSION TITLE
Lets create a list of services which required in all challanges

### DISCUSSION THREAD

**Discussion by L4stPL4Y3R** [BishopFox/cloudfox#62] [Ideas]
Hi Team,

There are many AWS sandbox service providers but the issue is what type of services are required to run all challenges. Some time its explicitly deny some specific service in playground. With the list of services are used in specific challenge get better understanding.

To create our own AWS account, it required debit/credit card for every new account.

It's just opinion, It will be great BishopFox can start providing AWS Sandbox for for all challenges with minimum cost :)

**Comment by sethsec-bf** [comment]
Thanks for the comment! I assume you are talking about for CloudFoxable, right?  For what it's worth, if you take one AWS account you have a credit card assigned to, and convert it into an organization management account, then you can spin up new AWS accounts within the organization for free and it all funnels to the single credit card!  Best of all, each new account you spin up gets the free tier, so you should incur very little cost, if any if you are simply using CloudFoxable! Let me know if this makes sense and is helpful.

**Reply by L4stPL4Y3R** [reply]
Thank you very much for the quick reply :) 

Yes, It's about CloudFoxable. I understand what you are saying :) we can make a list of services for each challenges :)

is there any discord channel for CloudFoxable so we can discuss with the community and help each other to build other new cloud pen testing challenges?

Thank you
